client/clientui.py

```diff
@@ -41,7 +41,8 @@ def __init__(self, master, client, queue, send_command):
         # pprint(vars(master))
         self.side_bar = master.children['side_bar']
         self.output_panel = master.children['output_frame'].children['output']
-        # self.output_panel = self.output_frame.children['output']
+        self.output_panel.configure(state="normal")
+        self.output_panel.bind('<Key>',lambda e: 'break')
```

Comment:
It creates a keybind for every key that calls the lambda function, which does nothing. It effectively blocks all keys from doing anything to the output_panel, which prevents us from having to toggle the configure state normal/disabled. I believe this toggling eventually causes the crashes.

We only need to do this once, so it seems like it should be with the rest of the output_panel initialization. I'd definitely like you to try it, as I saw some UI weirdness/duplication on the output_panel, but I'm not positive it was this change. If anything looks strange to you, I'll see if I can try out the other solution.